airbyte-oauth/src/main/java/io/airbyte/oauth/flows/MicrosoftTeamsOAuthFlow.java

```diff
@@ -129,46 +129,13 @@ private String getScopes() {
     throw new IOException("not supported");
```

Comment:
Can we be more specific on the error messaging please:
```suggestion
    throw new IOException("Using the deprecated OAuth methods is not supported. This OAuth flow depends on values defined in connector configs:", e);
```